src/Layers/xrRender/occRasterizer.cpp

```diff
@@ -69,10 +69,10 @@ occRasterizer::occRasterizer()
 occRasterizer::~occRasterizer() {}
 void occRasterizer::clear()
 {
-    u32 size = occ_dim * occ_dim;
+    size_t size = occ_dim * occ_dim;
     float f = 1.f;
     ZeroMemory(bufFrame, sizeof(bufFrame));
-    MemFill32(bufDepth, *LPDWORD(&f), size);
+    MemFill32(bufDepth, *(u32 *)(&f), size);
```

Comment:
static_cast will fill array integer numbers, but we need fill float value.

Reply:
reinterpret_cast then?

Reply:
The reinterpret_cast doesn't seem to be a preferred solution - it's not the device driver's code to use such 'hard' types of casts. Maybe it's better to write a cycle here to avoid magic with pointers and the potential problems in the future? I believe the compiler is smart enough to optimize the cycle in the binary.